### DISCUSSION TITLE
prisma migrate dev is taking a really long time these days

### DISCUSSION THREAD

**Discussion by amcjen** [prisma/prisma#9840] [Q&A]
We're running 3.2.0 and have 101 migrations. Whenever we run `prisma migrate dev` to create a new migration with changes to our schema, it's taking a really long time--like go-get-a-cup-of-coffee-and-enjoy-the-cup-of-coffee-before-it's-done-running long.

Screenshot showing the time it took for our most recent migrate (920 seconds).

Anything we can do to mitigate this? 

![Screen Shot 2021-10-18 at 3 22 19 PM](https://user-images.githubusercontent.com/175249/137815290-c085ef98-591d-4e9e-9e9b-939b924edd04.png)

**Comment by austincrim** [comment]
Hey @amcjen, thanks for bringing this to our attention. Would you mind converting this discussion to an issue and providing a bit more context? Any more you can tell us about your schema, the nature of your migrations, when the delays started to occur etc. would be really helpful.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
